tardis/tests/tests_slow/test_w7.py

```diff
@@ -55,32 +55,35 @@ def baseline(request):
 
     # Associate CGS units to ndarrays of baseline quantities.
     quantities['j_blues_norm_factor'] = \
```

Comment:
It's just a cosmetic thing but don't use `\` for line breaks. If it's a huge formula wrap it in braces, otherwise split after the function opening brace.
IE:
```
    quantities['last_line_interaction_angstrom'] = u.Quantity(
        quantities['last_line_interaction_angstrom'], 'Angstrom')
```